### ISSUE TITLE
After enabling this plugin, shared images cannot be displayed in a straight chain format.

### ISSUE THREAD

**Issue by WantSleep0419** [nextcloud/richdocuments#3877]
nextcloudV29
以下以访问`https://glxaa.com:11224/s/YSK6fXy5fwK5ZQ7/preview`为例说明：
启用后：
![image](https://github.com/user-attachments/assets/3fbd6797-ef93-4eed-9206-00fbf441d726)
显示
![image](https://github.com/user-attachments/assets/9004cd5b-8798-4ba2-9016-36f4cd52be0e)
禁用后正常显示：
![image](https://github.com/user-attachments/assets/2cd3bfd2-487a-4eaa-874e-3fb733cbbb27)

**Comment by WantSleep0419** [comment]
> 您能否用英文报告问题并填写问题模板？我无法理解您从标题中遇到的问题。

Simply put, this plugin has affected the display of image links in my area. That is to say` https://glxaa.com:11224/s/YSK6fXy5fwK5ZQ7 `The format allows browsing images on the web page, but when the suffix '/preview' is added, it displays as a '[]' and the image cannot be displayed properly.
When I disable this plugin, the images are linked in a straight chain format, with the suffix '/preview' added` https://glxaa.com:11224/s/YSK6fXy5fwK5ZQ7/preview `The image cannot be displayed.
At first, when I gradually upgraded from NextCloud27 to NextCloud29 (I had not used NextCloud/richdocuments before, but the plugin was enabled in the new version), I thought the problem was caused by the version upgrade. I raised a question in the community, but no one had the same situation. Afterwards, I rolled back to NextCloud27 and upgraded to NextCloud29 again. The problem of images not being displayed did not occur because I had not enabled the plugin. Finally, I tested back and forth several times and confirmed that it was' NextCloud/Richdocuments' that caused the image to not display in a straight chain format. Because many of the images in my MD files are written in a straight chain format in the document, this can cause me to be unable to display the image content properly in the MD file.
Attachment: When submitting this issue, I have already disabled the plugin, so the above URL can open the webpage normally.
